main.go

```diff
@@ -246,6 +246,20 @@ func serveCSS(c *gin.Context) {
 	serveFiles(c, "text/css", "./UI/styles/")
 }
 
+func requestTelemetry(c *gin.Context) {
+	c.Header("content-type", "application/json")
+	uri := fmt.Sprintf("http://%s:8080/send/", listIPs[4])
+
+	// Create JSON
+	json := "{\"verb\":\"GET\",\"uri\":\"http://" + listIPs[2] + ":8080/send/\"}"
+	body := strings.NewReader(json)
+
+	res, err := http.NewRequest("POST", uri, body)
```

Comment:
I am unsure as to what this does. I don't see:
- why you have written c.Header(application/json) when that is what is going to be in the response
- Where the return code for the function is

Reply:
1. That was unfortunately left over from another iteration of the function.
2. I goofed, and did not set the response status code.

Reply:
I should be solved in the latest commit